bundle/config/loader/process_include.go

```diff
@@ -3,12 +3,144 @@ package loader
 import (
 	"context"
 	"fmt"
+	"sort"
+	"strings"
 
 	"github.com/databricks/cli/bundle"
 	"github.com/databricks/cli/bundle/config"
 	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/dyn"
+	"golang.org/x/exp/maps"
 )
 
+var resourceTypes = []string{
+	"job",
+	"pipeline",
+	"model",
+	"experiment",
+	"model_serving_endpoint",
+	"registered_model",
+	"quality_monitor",
+	"schema",
+	"cluster",
+}
+
+func validateFileFormat(r *config.Root, filePath string) diag.Diagnostics {
+	for _, typ := range resourceTypes {
+		for _, ext := range []string{fmt.Sprintf(".%s.yml", typ), fmt.Sprintf(".%s.yaml", typ)} {
```

Comment:
I believe the plan is to have DABs in the workspace support both `.yml` and `.yaml` since asking users to change `.yaml` -> `.yml` is a bit arbitrary in a recommendation for DABs.

Reply:
Yeah, I agree.

@fjakobs Can you provide a snippet we can include for future readers wrt the expected extensions?

Reply:
We do recommend using `yml` but it feels a bit arbitrary to actively prohibit this.

It's a SHOULD in https://docs.google.com/document/d/1M4UqmNdpDgKMfuBVSghm_UtaDdYp4r-1SA0SuWsgG_k/edit#heading=h.bs9f118p0tig and not a must.